client/pages/AssessmentPrint/FraPrint/ContactPersons/ContactPersons.tsx

```diff
@@ -0,0 +1,64 @@
+import './ContactPersons.scss'
```

Comment:
maybe should be renamed into ContactPeople

Reply:
mmm or just Contacts if you really want to rename it

Reply:
I would leave as is, if you really want to rename, let me know!